_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
+continent, then country, then city.
+If we had reversed the order (city, then country, then continent),
+it would likely generate unintended results.
 
-##### Set threshold and severity levels
+###### Accounts to search
+
+![Accounts to search setting]({{site.baseurl}}/images/alerts/accounts-to-search.png)
+
+Next you'll choose the _Accounts to search_.
+
+If you choose **All accounts**,
+this alert will query the logs in all the accounts it has access to.
+This will include future accounts.
 
-Set your threshold and severity levels in the **Trigger** section.
-Click **Add multiple conditions** to add up to 3 threshold levels.
+To limit the alert to a set list of accounts,
+click **Just these accounts**,
+and add the accounts you want.
+This list is always limited to your selection,
+so you'll need to manually update the list if you need it to change.
+
+##### Set threshold and severity levels
 
 ![Alert trigger thresholds]({{site.baseurl}}/images/alerts/alerts--trigger-settings.png)
 
-##### _(Optional)_ Set recipients
+Set your threshold and severity levels in the _Trigger if..._ section.
+
+Click **Add a threshold** to set up to 5 threshold levels,
+each with its own severity tag.
+
+##### _(Optional)_ Set notification details
+
+###### Description and tags
+
+![Alert description and tags]({{site.baseurl}}/images/alerts/description-and-tags.png)
 
-If you want to receive notifications or emails when the alert is triggered,
-choose an alert endpoint.
-If you don't choose an endpoint,
-triggered alerts will still be logged:
+The **Description** is visible on the _Alert definitions_ page.
+It's also included with emails and Slack messages when the alert is triggered.
+We recommend making your description helpful to recipients,
+like telling them how to fix the issues that led to the alert.
 
-![Alert notification actions]({{site.baseurl}}/images/alerts/alerts--notification-action.png)
+The **Tags** are useful for filtering the _Alert definitions_ page.
 
-Choose the endpoints or email addresses to notify.
+###### Who to send it to
+
+![Recipients and suppress notifications]({{site.baseurl}}/images/alerts/recipients-and-suppress.png)
+
+If you want to send notifications or emails when the alert is triggered,
+choose notification endpoints.
+This isn't required, though—triggered alerts are still logged and searchable in Kibana.
+
+Choose the endpoints or email addresses to notify under _Who to send it to_.
 If you need help adding a new endpoint,
-see [Alert endpoints]({{site.baseurl}}/user-guide/integrations/endpoints.html).
+see [_Notification endpoints_]({{site.baseurl}}/user-guide/integrations/endpoints.html).
 
 Choose a time period to suppress notifications.
 
-When notifications are suppressed, Logz.io will continue to log triggered alerts without sending notifications. You can search triggered alert logs at any time.
+When notifications are suppressed,
+Logz.io will continue to log triggered alerts without sending notifications.
```

Comment:
Why is it in a note box? no need in my opinion

Reply:
This was from user feedback a while back. Apparently folks were confused about what suppressing notifications actually meant. I'll keep this as is unless there's a good reason to change.